Project Charter

GuillaumeDespaux opened the pull request:
There is the first version of the project charter.

GuillaumeDespaux committed:
Project Charter

Project Charter, need to be checked by the QA

GuillaumeDespaux committed:
Project Charter Update

Visual of arrays more clear + grammatical errors fixed

Enonome reviewed (changes requested):
I don't know if it's really relevant but, on the 4th, 5th and 17th lines you can put a "s" (plural) on "movement**s**", "integration**s**", "functionalit**ies**" and "performance**s**" respectively.
- Also, on the 16th and 17th lines, the 2 software engineers don't have the same contribution area.
- Moreover, you can migrate *milestones* in the *deliverables* (to avoid repetitions).
- Finally, according to the Skills schedule, I have the impression that we have only 13 days for the total half-days of work, so, I think the estimated human hours is not correct (maybe 45h and 30 minutes idk)

GuillaumeDespaux committed:
Correction of Project Charter

Correction due to review
https://github.com/algosup/2024-2025-project-1-fpga-team-2/pull/1#pullrequestreview-2333587497

GuillaumeDespaux committed:
Implementation Milestones removal

Enonome reviewed (approved):
Okay for me
